app/config.go

```diff
@@ -74,7 +74,7 @@ const (
 	PlaceholderHash                 = "00"
 	PlaceholderURL                  = "https://foo.bar:8080"
 	PlaceholderServiceURL           = PlaceholderURL
-	DefaultRemoteCLIURL             = "http://localhost"
+	DefaultRemoteCLIURL             = "http://localhost:8081"
```

Comment:
Let's leave this as localhost and append the RPCPort variable during usage instead

Reply:
@andrewnguyen22 this would work for localhost usage but we don't have control over the RPC port the remote CLI is using at execution time.

Reply:
Good call!